masonry/src/tracing_backend.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(unused)]
```

Comment:
Can we have a comment here? I'm guessing this is because if the event loop runner is conditionally compiled out, this is unused?